.github/workflows/ci.yml

```diff
@@ -13,13 +13,12 @@ jobs:
 
     steps:
       - name: Set up PHP
-        uses: shivammathur/setup-php@1.7.0
+        uses: shivammathur/setup-php@2.3.1
```

Comment:
Could we make this just `@v2` like we did for checkout and node? I don't think we need to pin?

Reply:
Actually, that is not the same.. 

Github actions maintainers are suggested to update the tag. 

So `shivammathur/setup-php@2.3.1` means "get version 2.3.1. 
And `shivammathur/setup-php@v2` means "get version v2. 

It is nothing like composer. If maintainers of `shivammathur/setup-php` does not update (move) the tag, then we are stuck with an old version... Just like now.. 

With this we know what version we are using. If a future version includes stuff that we like, then we just change the version here.

Reply:
v2 does actually seem to get 2.3.1, otherwise the composer:v2 thing wouldn't work.